pr_check.sh

```diff
@@ -1,12 +1,22 @@
 #!/bin/bash
 
-set -exv
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="compliance"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="ingress"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/compliance-backend"
 
-# Make directory for artifacts
-mkdir -p artifacts
+IQE_PLUGINS="compliance"
+IQE_MARKER_EXPRESSION="smoke"
+IQE_FILTER_EXPRESSION=""
 
-cat << EOF > artifacts/junit-dummy.xml
-<testsuite tests="1">
-    <testcase classname="dummy" name="dummytest"/>
-</testsuite>
-EOF
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
```

Comment:
Even-thought this is an example copy, can we use the same sha2 checking as we have for other shared bash files, pinning it to a specific version?
This opens up a door from the bonfire repo into our pr_check.

Reply:
I went looking into this, and I was planning to make a local copy, but the last line of [bootstrap.sh](https://github.com/RedHatInsights/bonfire/blob/master/cicd/bootstrap.sh#L24) is:

```
git clone https://github.com/RedHatInsights/bonfire.git
```

Maybe we can accept this for now and work with bonfire to make it more secure, or we could clone a specific SHA and run `./cicd/bootstrap.sh`. Thoughts?